src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/chatpageobject/ChatPageObject.java

```diff
@@ -243,6 +267,69 @@ public void banUser(String userName) {
     PageObjectLogging.log("clickBanUser", userName + " ban modal is closed", true);
   }
 
+  public void clickOpenUserMessageWall(String userName) {
```

Comment:
In this one and those around here I'd assume that if I pass userName it will ensure that it will open a message wall page for a given user. Looking at the code the userName is only used to add in the logs which apparently may be misleading when it opens message wall page for a different user for whatever reason.

Am I missing something?